aten/src/ATen/WrapDimUtils.h

```diff
@@ -72,4 +72,13 @@ static inline int64_t legacy_cat_wrap_dim(int64_t dim, TensorList tensors) {
   return dim;
 }
 
+// wrap negative dims in v, where total dim is n
+static inline void wrap_dims(std::vector<int64_t>& v, int64_t n) {
```

Comment:
can this have similar signature as the other functions, i.e., having `int64_t dim` as first arg and more descriptive arg names?

Reply:
Sure, I will rename the variables.